### PR TITLE
Adding `concurrent-start/end` for Concurrent Global/Marking

### PR DESCRIPTION
**Pull request by gza060625**
This is follow up PR for https://github.com/eclipse/omr/pull/5781

Adapt `MM_VerboseHandlerOutputVLHGC::getConcurrentTypeString` to interface of parent class.

Signed-off-by: Enson Guo <enson.guo@ibm.com>

### CODE REVIEW

File: runtime/gc_verbose_handler_vlhgc/VerboseHandlerOutputVLHGC.hpp
Comment (RSalman) [inline runtime/gc_verbose_handler_vlhgc/VerboseHandlerOutputVLHGC.hpp:159]: With this removed, there's a complication on relying on `getCycleType(_stats->concurrentType)` for the concurrent type string. 
`OMR_GC_CYCLE_TYPE_VLHGC_GLOBAL_MARK_PHASE` will result in "global mark phase" (see MM_VerboseHandlerOutputVLHGC::getCycleType)  and according to this original implementation it should be "GMP work packet processing"

**Review by RSalman — CHANGES_REQUESTED** [review]

**Review by RSalman — COMMENTED** [review]
LGTM - will approve once dependent changes are finalized/approved

**Review by RSalman — APPROVED** [review]

**Comment by RSalman** [comment]
@amicic Ready to be merged, OMR changes are in.

**Comment by amicic** [comment]
This is mutually dependent on/to https://github.com/eclipse/omr/pull/5781 (which is merged, but not promoted), so not make sense even trying to compile. Merging....

**Pull request merged**
